src/game/spawn/mod.rs

```diff
@@ -0,0 +1,11 @@
+//! Handles spawning of entities. This is typically done through
+//! [observers](https://docs.rs/bevy/latest/bevy/ecs/prelude/struct.Observer.html).
```

Comment:
```suggestion
//! [observers](https://docs.rs/bevy/latest/bevy/ecs/prelude/struct.Observer.html).
// Alternatively this can be done through `Events<E>` or `Commands`.
```